cmd/estuary-shuttle/main.go

```diff
@@ -275,7 +287,8 @@ func main() {
 		},
 		&cli.BoolFlag{
 			Name:  "libp2p-websockets",
-			Value: cfg.NodeConfig.HasListener(config.DefaultWebsocketAddr),
+			Usage: "enable adding libp2p websockets listen addr",
```

Comment:
Another question: what does this mean? Can we make the wording more intuitive?

Reply:
This checks if the libp2p websockets addr should be added to the listen addrs set.